buildpacks/metrics-agent/buildpack.toml

```diff
@@ -0,0 +1,23 @@
+api = "0.9"
+
+[buildpack]
+id = "heroku/statsd-metrics"
+version = "2.0.0"
+name = "Statsd Metrics"
+homepage = "https://github.com/heroku/buildpacks-ruby"
+description = "Installs an agent to send language specific metrics back to heroku via statsd"
+keywords = ["statsd", "metrics"]
+
+[[stacks]]
+id = "heroku-20"
+
+[[stacks]]
+id = "heroku-22"
+
+[[buildpack.licenses]]
+type = "BSD-3-Clause"
+
+[metadata]
+[metadata.release]
+[metadata.release.docker]
```

Comment:
the key has been renamed from `docker` to `image`

Reply:
Thanks for the heads up. I also need to make a docker registry for this too.